.github/workflows/mmv1-check-templates.yml

```diff
@@ -0,0 +1,30 @@
+name: mmv1-check-templates
+
+permissions: read-all
+
+on:
+  pull_request:
+    paths:
+      - 'mmv1/**/*.erb'
+
+jobs:
+  version-guard-check:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
```

Comment:
I think we'll want to pin the version by a specific hash. Looks like that's how we set up for other workflows after https://github.com/GoogleCloudPlatform/magic-modules/pull/10239

Reply:
Nice catch! My copy-paste missed that change, should be fixed now.